docs/modules/maintain-cluster/pages/rolling-upgrades.adoc

```diff
@@ -48,8 +60,23 @@ As a result, you can use rolling upgrades to upgrade clusters to the following m
 +
 This also includes the ability of rolling upgrade from Hazelcast IMDG 4.2 to Platform 5.0, since IMDG is part of the Platform and 5.0 is considered to be the next minor version after IMDG 4.2.
 
+- Releases from 5.2 to 5.4, directly to the 5.5 Long-term Support (LTS) release. For further information on additional compatibilities for direct-to-LTS rolling upgrades, see xref:migrate:lts.adoc[].
```

Comment:
```suggestion
- Directly to the 5.5 Long-term Support (LTS) release from releases 5.2 through 5.4. For further information on additional compatibilities for direct-to-LTS rolling upgrades, see xref:migrate:lts.adoc[].
```
Reads a little awkwardly with the source releases at front, just a suggestion...

Reply:
Changed to: The 5.5 Long-term Support (LTS) release direct from 5.2.x to 5.4.x.

Which is actually better for list parallelism as well as readability! Good catch.